src/OpenTelemetry.Exporter.Console/ConsoleLogRecordExporter.cs

```diff
@@ -129,5 +154,22 @@ void ProcessScope(LogRecordScope scope, ConsoleLogRecordExporter exporter)
 
             return ExportResult.Success;
         }
+
+        protected override void Dispose(bool disposing)
+        {
+            if (!this.disposed)
```

Comment:
I don't think we need this. Dispose is never supposed to be called concurrently - if that ever happened, it is a bigger issue that should be handled at a different level, not here.

Reply:
updated, removed the lock